### PR TITLE
Firebase docs

### PR DESCRIPTION
**Pull request by 1abhishekpandey**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: destinations/analytics/firebase.mdx [hunk 1]
```diff
@@ -469,62 +468,66 @@ The `identify` call from RudderStack sets the `userId` through `setUserId` metho
...
-| `price`                | `price`                |
+| Standard Rudder Name        | Standard Firebase Name   |
+| :-------------------------- | :---------------------   |
+| `category`                  | `item_category`          |
+| `cart_id`,`product_id`      | `item_id`                |
+| `share_via`                 | `method`                 |
+| `query`                     | `search_term`            |
+| `revenue`, `value`, `total` | `value`                  |
+| `currency`                  | `currency`               |
+| `order_id`                  | `transaction_id`         |
```
Comment (pallabmaiti) [inline destinations/analytics/firebase.mdx:518]: @1abhishekpandey  Transaction id shouldn't be order id. Did we implement like this?

File: destinations/analytics/firebase.mdx [hunk 2]
```diff
@@ -546,7 +549,48 @@ A sample `track` call from the iOS SDK will look like the below
...
+                .putValue("Width",12)
+        )
+```
+{% endtab %}
+{% endtabs %}
+
+
+# Reset
+
+The `reset` method calls `setUserId(null)` which removes the user ID.
```
Comment (pallabmaiti) [inline destinations/analytics/firebase.mdx:579]: @1abhishekpandey no need to mention setUserId(null)

**Review by pallabmaiti — CHANGES_REQUESTED** [review]

File: destinations/analytics/firebase.mdx (hunk 1 above)
Comment (1abhishekpandey) [reply destinations/analytics/firebase.mdx:519]: Yes, it was implemented like this. I haven't modified this mapping. Please let me know if any changes are required w.r.t this.

Comment (pallabmaiti) [reply destinations/analytics/firebase.mdx:519]: I think we shouldn't map transaction_id

**Review by pallabmaiti — DISMISSED** [review]

**Review by ameypv-rudder — APPROVED** [review]
Good to go!

**Review by arnab-p — APPROVED** [review]

**Pull request merged**